Missing Signatures After the comp (#59)
CarterKoala/wca-regulations-wrc: wca-regulations.md

        - A7c2) If the competitor intentionally signs (or otherwise marks) an attempt before they have begun it, they forfeit the attempt and the result will be considered DNS.
        - A7c3) If the competitor signs (or otherwise marks) an attempt after starting the attempt but before the judge has finished recording the result, the attempt will be considered unfinished (DNF).
        - A7c4) The competitor should not sign an attempt before the judge has recorded and signed the attempt. The competitor is ultimately responsible for ensuring that they and the judge have signed an attempt. If an attempt is found to be missing one or both signatures after the judge delivered the score sheet to the score taker (see [Regulation A7f](regulations:regulation:A7f)), it will be considered unfinished (DNF).

>         - A7c5) Missing signatures found before the end of the competition (see [Regulation 9u](regulations:regulation:9u)) may be added by the judge and/or the competitor, at the discretion of the WCA Delegate. If the judge and/or competitor are no longer able to physically sign the scorecard, because they have left the competition, or the competition has finished, their attestation that the result was correctly recorded may be accepted as a signature.

Reviewer: ```suggestion
        - A7c5) Missing signatures found before the end of the competition (see [Regulation 9u](regulations:regulation:9u)) may be added by the judge and/or the competitor, at the discretion of the WCA Delegate. Exception: if a signature was missed due to the inexperience of the competitor, it may be added after the competition has ended, at the discretion of the WCA Delegate.  If the judge and/or competitor are no longer able to physically sign the scorecard, because they have left the competition, or the competition has finished, their attestation that the result was correctly recorded may be accepted as a signature.
```
Something like this? (And we probably want to split out the exception into a sub-reg and the non-physical signatures part into a guideline).

Author: ```suggestion
        - A7c5) Missing signatures found before the end of the competition (see [Regulation 9u](regulations:regulation:9u)) may be added by the judge and/or the competitor, at the discretion of the WCA Delegate. Exception: if a signature was missed due to the inexperience of the competitor and it is no longer possible for the scorecard to be physically signed, an attestation that the result was correctly recorded may be accepted as a signature after the competition, at the discretion of the WCA Delegate.
```
We run out of sub-regs so we can't really add it there, and I think the non physical signatures part needs to be part of the regs rather than the guidelines. This suggestion simplifies the wording a little bit.

Author: Moved this to A7c6 instead

        - A7c3) If the competitor signs (or otherwise marks) an attempt after starting the attempt but before the judge has finished recording the result, the attempt will be considered unfinished (DNF).
        - A7c4) The competitor should not sign an attempt before the judge has recorded and signed the attempt. The competitor is ultimately responsible for ensuring that they and the judge have signed an attempt. If an attempt is found to be missing one or both signatures after the judge delivered the score sheet to the score taker (see [Regulation A7f](regulations:regulation:A7f)), it will be considered unfinished (DNF).
        - A7c5) Missing signatures found before the end of the competition (see [Regulation 9u](regulations:regulation:9u)) may be added by the judge and/or the competitor, at the discretion of the WCA Delegate.

>         - A7c6) If a signature was missed due to the inexperience of the competitor and it is no longer possible for the scorecard to be physically signed by the competitor or the judge, an attestation that the result was correctly recorded may be accepted as a signature after the competition, at the discretion of the WCA Delegate.

Reviewer: ```suggestion
        - A7c6) If a signature was missed due to the inexperience of the competitor, it may be added at any time, at the discretion of the WCA Delegate. If is no longer possible for the scorecard to be physically signed by the competitor or the judge, an attestation that the result was correctly recorded may be accepted as a signature, at the discretion of the WCA Delegate.
```

>         - A7c6) If a signature was missed due to the inexperience of the competitor, it may be added at any time, at the discretion of the WCA Delegate. If is no longer possible for the scorecard to be physically signed by the competitor or the judge, an attestation that the result was correctly recorded may be accepted as a signature, at the discretion of the WCA Delegate.

Author: ```suggestion
        - A7c6) If a signature was missed due to the inexperience of the competitor, it may be added at any time, at the discretion of the WCA Delegate. If it is no longer possible for the scorecard to be physically signed by the competitor or the judge, an attestation that the result was correctly recorded may be accepted as a signature, at the discretion of the WCA Delegate.
```
